apps/expo/src/components/NavBar.tsx

```diff
@@ -73,6 +73,13 @@ const NavBar = () => {
           tabBarLabel: "Account",
         }}
       />
+      <Tabs.Screen
+        name="signin"
+        options={{
+          href: "/signin",
+          tabBarLabel: "Sign In",
+        }}
+      />
```

Comment:
Instead of creating a Tab, I think that you want to wrap the NavBar in _layout.tsx with the SignedIn component: https://clerk.com/docs/quickstarts/expo#protecting-your-pages